product-details.html

```diff
@@ -0,0 +1,459 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <nav class="navbar">
+      <a href="./index.html" class="nav-link">
+        <img src="#" alt="main-logo" class="main-logo" />
+      </a>
+      <ul class="nav-list">
+        <li class="nav-item">
+          <a href="./about.html" class="nav-link">About</a>
+        </li>
+        <li class="nav-item">
+          <a href="./products.html" class="nav-link">Products</a>
+        </li>
+        <li class="nav-item">
+          <a href="./creators.html" class="nav-link">
+            Creators <span class="nav-new-highlight">new</span>
+          </a>
+        </li>
+        <li class="nav-item">
+          <a href="./blog.html" class="nav-link">Blog</a>
+        </li>
+        <li class="nav-item">
+          <a href="./contact.html" class="nav-link">Contact</a>
+        </li>
+      </ul>
+      <div class="nav-account-options">
+        <a href="#" class="nav-account-item">
+          <img src="#" alt="profileIcon" class="profile-icon" />
+        </a>
+        <a href="#" class="nav-account-link">
+          <img src="#" alt="cartIcon" class="cart-icon" />
+        </a>
+      </div>
+    </nav>
+    <main class="main-container">
+      <section class="product-container">
+        <div class="product-left">
+          <figure class="active-image-illustration">
+            <img src="#" alt="ProductImg" class="active-product-image" />
+            <button class="expand-img-btn">
+              <img src="#" alt="ExpandBtn" />
+            </button>
+          </figure>
+          <div class="product-img-selector">
```

Comment:
This block can be a part of figure tag, because figure also using for group images.